presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -83,6 +83,9 @@
     @Option(name = "--access-token", title = "access token", description = "Access token")
     public String accessToken;
 
+    @Option(name = "--delegation-token", title = "delegation token", description = "HDFS delegation token")
```

Comment:
Should we consider using the `extraCredentials` mechanism to accomplish this, rather than introduce a new HDFS specific option here?